libvast/src/system/importer.cpp

```diff
@@ -40,23 +40,33 @@ namespace vast::system {
 
 namespace {
 
-class driver
-  : public caf::stream_stage_driver<
-      table_slice,
-      caf::broadcast_downstream_manager<detail::framed<table_slice>>> {
+class driver : public caf::stream_stage_driver<
+                 table_slice, caf::broadcast_downstream_manager<table_slice>> {
 public:
-  driver(caf::broadcast_downstream_manager<detail::framed<table_slice>>& out,
+  driver(caf::broadcast_downstream_manager<table_slice>& out,
          importer_state& state)
     : stream_stage_driver(out), state{state} {
     // nop
   }
 
-  void process(caf::downstream<detail::framed<table_slice>>& out,
+  void process(caf::downstream<table_slice>& out,
                std::vector<table_slice>& slices) override {
     VAST_TRACE_SCOPE("{}", VAST_ARG(slices));
     uint64_t events = 0;
     auto t = timer::start(state.measurement_);
     for (auto&& slice : std::exchange(slices, {})) {
+      auto import_ok = state.executor.add(std::move(slice));
+      if (import_ok) {
+        VAST_WARN("importer can not add slice to pipeline executor - data "
```

Comment:
Could we also print the import_ok error here?

Reply:
I added an `import_ok` print in https://github.com/tenzir/vast/pull/2896/commits/e9bfda4fb7575c59b37e284cf8b05d763d403c7e

Reply:
`import_ok` implies the opposite meaning of what it actually carries. As a reader I'm very confused by line 59 which I can only parse as "is the import ok? then print a warning".
How about changing this to
```
if (auto err = state.executor.add(std::move(slice))
  VAST_ERROR(...);
```?

Reply:
That's true - now I wonder where that name originally came from... probably from when I thought `executor::add()` would return a `caf::expected` instead of a `caf::error`. I renamed the variable in https://github.com/tenzir/vast/pull/2896/commits/1cf34b766a041ca1349e717b5121fc397061c2f3